File: cpp/template.cpp
#include <iostream>
#include <chrono>
#include "utils.h"

int64_t compute()
{
    return 0;
}

int test_prob000() {
    std::string answer;
    get_answer(1, &answer);

    assert(answer.compare("1234") == 0);
    return 0;
}

int main(int argc, char const *argv[])
{
    auto t1 = std::chrono::high_resolution_clock::now();
    int64_t result = compute();
    auto t2 = std::chrono::high_resolution_clock::now();

    auto duration = std::chrono::duration_cast<std::chrono::microseconds>(t2 - t1).count();
    std::cout << result << "\tin " << duration << " us\n";
    return test_prob000();
}

File: py/template.py
import os
import csv
import time

PROB_NUM = 1

def compute():
    return 233168


def test_prob000():
    dir_path = os.path.dirname(__file__)
    answer_file = open(os.path.join(dir_path, "../answers.txt"))
    answers = list(csv.reader(answer_file, delimiter='\n'))

    assert(str(compute()) == answers[PROB_NUM][0][4:])

if __name__ == "__main__":
    now = time.perf_counter()
    result = compute()
    duration = time.perf_counter() - now
    
    print("%d in %dms" % (result, duration))